cmd/bundle/summary.go

```diff
@@ -0,0 +1,67 @@
+package bundle
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/deploy/terraform"
+
+	"github.com/databricks/cli/bundle/phases"
+	"github.com/spf13/cobra"
+)
+
+func newSummaryCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "summary",
+		Short: "Describe the bundle resources and their deployment states",
+
+		PreRunE: ConfigureBundleWithVariables,
+
+		// This command is currently intended for the Databricks VSCode extension only
+		Hidden: true,
+	}
+
+	var forcePull bool
+	cmd.Flags().BoolVar(&forcePull, "force-pull", false, "Skip local cache and load the state from the remote workspace")
```

Comment:
Do you use this for vscode?

Reply:
Yeah. It'll allow caching the state and refreshing the state occasionally.